### ISSUE TITLE
Accessibility Testing for [Facilities, VAMC, Real-time banners]

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-team#97684]
### Product information

- [x] Team name, product name, and feature name have been added to the title of this issue.
- [x] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

@laflannery 

### Use of color and color contrast checks

- [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [x] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [x] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [x] Any charts, maps, infographics, and tables convey all information without only relying on color
- [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?

Great

### Who completed the axe scans?

@laflannery 

### axe checks

- [x] Each page has been scanned using axe (results shared below)
- [ ] axe is integrated into your end-to-end testing
   - We are unfortunately unable to do e2e testing with this product

### axe DevTools scan results

Axe clean!
![Image](https://github.com/user-attachments/assets/eda5a6b0-f7b4-4f88-a6a7-97d1d51448a9)

### How did axe testing go?

Wonderfully

### Who completed the content zoom and reflow test?

@laflannery 

### Content zoom and reflow checks

- [x] All page elements are readable and usable at 200% zoom
- [x] All page elements are readable and usable at 300% zoom
- [x] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?

Excellent
### Who completed the keyboard navigation test?

@laflannery 

### Keyboard navigation checks

- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [x] All elements under focus have a visible focus indicator
- [ ] The order of [Tab] stops made sense and was appropriate for completing tasks
  - I am not totally sold on the focus order of the close button relative to the rest of the content. I would like others opinions (added a question for other specialists) and depending on responses I can file a ticket for DST

### How did keyboard testing go?

Good
### Do you have any other results to share?

I [filed a bug with DST](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3594) because I noticed that the close button is very far right on large screens and I feel that this may get missed by some users, especially screen magnifier users.

**Comment by jilladams** [comment]
Creating this ticket early so we can task filling it out in our upcoming sprint. @briandeconinck is it ok with you if we reassign it to my team and track it in our sprint, and reassign to you when we're done / ready for review?  (Our staging review is currently scheduled for 12/20, but we're trying to pull it in earlier.)

**Comment by laflannery** [comment]
I'm reopening because I can't remember the process and if I should have when I was done or not

**Comment by jilladams** [comment]
@briandeconinck this is now ready for your review, when you get to this project. (I think Shira has time booked for you on Wed 12/11 1-5p ET.)

**Comment by jilladams** [comment]
We passed staging review. Closing. Reopen as needed.
